ice40/smoketest/attosoc/smoketest.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+rm -f picorv32.v
+wget https://raw.githubusercontent.com/cliffordwolf/picorv32/master/picorv32.v
```

Comment:
Please vendor `picorv32.v` into the smoketest (either into the repo itself or the Docker image), we don't want to repeatedly wget from GitHub.